app/controllers/modified useremail function to include banned users

```diff
@@ -65,15 +65,21 @@ def useremail
       if params[:address]
         # address was submitted. find the username(s) and return.
         @address = params[:address]
-        @users = User.where(email: params[:address])
-                 .where(status: [1, 4])
+        if params[:include_banned]
+          @users = User.where(email: params[:address]) 
+          .where('created_at > (?)', DateTime.new(2015)) # since 2015, whether banned or not
+        else
+          @users = User.where(email: params[:address]) 
+          .where(status: [1, 4]) 
+        end
+      else
+        # unauthorized. instead of return ugly 403, just send somewhere else
+        redirect_to '/dashboard'
       end
```

Comment:
Oh, take a look here. Is this `end` improperly nested? Shouldn't the `redirect_to` go after it?

Reply:
I've done the changes. Please have a look @jywarren